src/transaction/boot_cl.c

```diff
@@ -1305,6 +1305,13 @@ boot_restart_client (BOOT_CLIENT_CREDENTIAL * client_credential)
     }
   json_set_alloc_funcs (malloc, free);
 
+  /* Initialize of the lzo library. */
+  error_code = lzo_init ();
+  if (error_code != LZO_E_OK)
+    {
+      goto error;
+    }
+
```

Comment:
@eseokoh
We felt like we needed to add the initialize of the LZO library on the client aswell, since on the server is already done. What do you think about it?